common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -239,6 +244,11 @@
           "Error reading streaming state file of <fileToRead> does not exist. If the stream job is restarted with a new or updated state operation, please create a new checkpoint location or clear the existing checkpoint location."
         ]
       },
+      "SNAPSHOT_PARTITION_ID_NOT_FOUND" : {
+        "message" : [
+          "Partition id <snapshotPartitionId> not found for given state source."
```

Comment:
"given state source" might not give enough info. Can we also add some additional metadata such as checkpoint location, operator id etc ?